packages/elements/src/combo-box/helpers/renderer.ts

```diff
@@ -1,30 +1,43 @@
-import type { CollectionComposer } from '@refinitiv-ui/utils/collection.js';
+import { DeprecationNotice } from '@refinitiv-ui/core';
+
+import type { CollectionComposer, DataItem } from '@refinitiv-ui/utils/collection.js';
 
 import type { Item, ItemData } from '../../item';
-import { ListRenderer } from '../../list/index.js';
+import { createListRenderer } from '../../list/index.js';
 import { Renderer } from '../../list/renderer.js';
 
+export const createComboBoxRenderer = <T extends DataItem = ItemData>(
+  context?: unknown
+): ((item: T, composer: CollectionComposer<T>, element?: HTMLElement) => HTMLElement) => {
```

Comment:
This looks unnecessary.
```suggestion
) => {
```